jobs/ci_open_jenkins/build/cgt.groovy

```diff
@@ -17,3 +17,8 @@ new SbtMicroserviceJobBuilder('capital-gains-calculator').
 
 new BuildMonitorViewBuilder('CGT-MONITOR')
```

Comment:
Do we need an entry in here for the CGT-MONITOR?

Or do we need a new BuildMonitorViewBuilder?